mlflow/store/tracking/file_store.py

```diff
@@ -1940,7 +1942,7 @@ def _list_trace_infos(self, experiment_id):
 
     def create_logged_model(
         self,
-        experiment_id: str,
+        experiment_id: Optional[str] = None,
```

Comment:
`experiment_id` should not be None.

Reply:
Shouldn't we use default experiment_id if it's None? Like below implementation `experiment_id = FileStore.DEFAULT_EXPERIMENT_ID if experiment_id is None else experiment_id`

Reply:
We can use the default experiment but that's not FileStore's job. :)